__init__.py

```diff
@@ -0,0 +1,5 @@
+
+
+from .src import program
+from .src.parse import parse
+from .src.passes.passes import Pass
```

Comment:
new line

Also you clearly know how to manage python projects better than me xD